app/controllers/ems_infra_controller.rb

```diff
@@ -33,52 +33,79 @@ def scaling
       return
     end
 
-    @count_parameters = @stack.parameters.select { |x| x.name.include?('::count') }
+    arel_table = OrchestrationStackParameter.arel_table
+    @orig_parameters = @stack.parameters.where(arel_table[:name].matches("%::count"))
```

Comment:
Ladas changed it in another PR to get it directly from the stacks. No more arel_table.